lib/ohai.rb

```diff
@@ -17,6 +17,5 @@
 #
 
 require "ohai/version"
-require "ohai/config"
```

Comment:
We can keep it here as well, but it's getting included with Application still:

Application -> Ohai -> System -> Config